cross/hoverkite-firmware/src/hoverboard.rs

```diff
@@ -429,4 +430,21 @@ impl Hoverboard {
             shared.motor.position = 0;
         })
     }
+
+    /// Set the motor PWM values directly for testing.
+    pub fn set_motor_pwm_for_test(&mut self, y_percent: u8, b_percent: u8, g_percent: u8) {
+        free(|cs| {
+            // SHARED must have been initialised by the time this is called.
+            let shared = &mut *SHARED.borrow(cs).borrow_mut();
```

Comment:
what is this `&mut *` reborrow doing?

Reply:
Turning a `RefMut<Option<Shared>>` into an `&mut Option<Shared>`.